Source/qol/stash.cpp

```diff
@@ -453,7 +407,8 @@ bool UseStashItem(uint16_t c)
 	if (stextflag != STORE_NONE)
 		return true;
 
-	Item *item = &Stash.stashList[c];
+	Item *item = &Stash.stashList[itemIndex];
+	Point itemPosition = item->position;

```

Comment:
Nothing modifies item->position between this line and when the call to `Stash.RemoveStashItem()` occurs on line 459. This variable can be inlined or the declaration moved to before the first call that potentially modifies the position (if I've missed it).

Reply:
Suggestion applied in most recent push